src/sentry/conf/server.py

```diff
@@ -31,7 +31,6 @@
 from sentry.conf.types.service_options import ServiceOptions
 from sentry.conf.types.taskworker import ScheduleConfigMap
 from sentry.conf.types.uptime import UptimeRegionConfig
-from sentry.utils import json  # NOQA (used in getsentry config)
```

Comment:
I don't think this should be removed.

Reply:
This happened in my other PR too. I think vscode is removing it on save :(